notebooks/ImvizExample.ipynb

```diff
@@ -189,6 +189,34 @@
     "imviz.app"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "db7b63f1-ccbb-45a5-8938-961ab207cfe6",
+   "metadata": {},
+   "source": [
+    "The above will show imviz inline in the notebook.  The following two cells are alternative ways to show it in separate windows, either on the right side of the notebook or in a new tab.  (Note this will only work in Jupyterlab, *not* the \"classic\" notebook.)"
```

Comment:
```suggestion
    "The above will show Imviz inline in the notebook.  The following two cells are alternative ways to show it in separate windows, either on the right side of the notebook or in a new tab.  (Note this will only work in JupyterLab, *not* the \"classic\" notebook.)"
```